angular/src/pipes/credit-card-number.pipe.ts

```diff
@@ -0,0 +1,197 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({ name: 'creditCardNumber' })
+export class CreditCardNumberPipe implements PipeTransform {
+
+    numberFormats = {
```

Comment:
Since we don't plan on modifying these lets move them outside the class as a `const`.